Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/NodeRecord.php

```diff
@@ -107,7 +125,22 @@ public static function fromDatabaseRow(array $databaseRow): self
             SerializedPropertyValues::fromArray(json_decode($databaseRow['properties'], true)),
             NodeTypeName::fromString($databaseRow['nodetypename']),
             NodeAggregateClassification::from($databaseRow['classification']),
-            isset($databaseRow['name']) ? NodeName::fromString($databaseRow['name']) : null
+            isset($databaseRow['name']) ? NodeName::fromString($databaseRow['name']) : null,
+            Timestamps::create(
+                self::parseDateTimeString($databaseRow['created']),
+                self::parseDateTimeString($databaseRow['originalcreated']),
```

Comment:
this makes the change breaking, doesn't it? (in an existing instance). I'd like to discuss whether we can make it non breaking :)

Reply:
breaking because it requires a `./flow cr:setup` or why do you mean? Ah the projection replay.. Mh, yes, lets talk about it!

Reply:
breaking, if created is null - exactly :)